sound/soc/sof/intel/tgl.c

```diff
@@ -76,6 +78,20 @@ int sof_tgl_ops_init(struct snd_sof_dev *sdev)
 	/* ipc */
 	sof_tgl_ops.send_msg	= cnl_ipc_send_msg;
 
+	/* set DAI ops */
+	for (i = 0; i < sof_tgl_ops.num_drv; i++) {
+		if (strstr(sof_tgl_ops.drv[i].name, "SSP")) {
+			sof_tgl_ops.drv[i].ops = &ipc3_ssp_dai_ops;
+			continue;
+		}
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
+		if (strstr(sof_tgl_ops.drv[i].name, "iDisp") ||
+		    strstr(sof_tgl_ops.drv[i].name, "Analog") ||
+		    strstr(sof_tgl_ops.drv[i].name, "Digital"))
+			sof_tgl_ops.drv[i].ops = &ipc3_hda_link_dai_ops;
+#endif
+	}
+
```

Comment:
I would make this a helper function, that way we avoid the duplication of code and if we need an additional interface to be configured it's done for all platforms.

Reply:
do you want me to update this PR or do it in a follow up PR @plbossart ?

Reply:
probably better to update this PR and do it once?

Reply:
done!